### DISCUSSION TITLE
Case for Wlanpi M4?

### DISCUSSION THREAD

**Discussion by nimbo78** [WLAN-Pi/feedback#49] [Hardware - WLAN Pi M4] [answered]
will STL be available for Wlanpi M4 like it is for Wlanpi R4? (fascia)

**Comment by nickjvturner** [comment, marked as answer]
Hi there, currently I do not plan on releasing the model files for the WLAN Pi M4.
I want to support makers who would like to assemble their own WLAN Pi M4.
I am happy to sell you a case with all 3D printed parts, let me know if that is of interest.

**Reply by who-1s-th1s** [reply]
Hey Nick! Is there a possibility that we buy the STLs for the case? Thanks!
